docs/eventing/getting-started.md

```diff
@@ -68,6 +68,7 @@ demonstrate how you can configure your event producers to target a specific cons
      kind: Deployment
      metadata:
        name: hello-display
+       namespace: event-example
```

Comment:
The namespace is being handled in the command line `kubectl -n event-example` do not add in the yaml. 
Doing in the command line is more flexible as allows the resource to be deployed to another namespace.
/hold

Reply:
ok，thanks